src/System.Private.CoreLib/src/System/GC.cs

```diff
@@ -78,6 +78,17 @@ public static GCMemoryInfo GetGCMemoryInfo()
                                     fragmentedBytes: (long)(ulong)lastRecordedFragmentationBytes);
         }
 
+        [DllImport(RuntimeHelpers.QCall, CharSet = CharSet.Unicode)]
+        internal static extern void GetGCConfigValue(string name, StringHandleOnStack retString);
+
+        // internal for now, for testing that configuration works
+        internal static string GetConfigValue(string name)
```

Comment:
>GetConfigValue [](start = 31, length = 14)

Andy, can you please file an API proposal for this? for the next release we should provide this as a public API

Reply:
dotnet/corefx#42499